client/src/main/java/com/vesoft/nebula/client/storage/scan/PartScanInfo.java

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.storage.scan;
+
+import com.vesoft.nebula.client.graph.data.HostAddress;
+import java.io.Serializable;
+
+public class PartScanInfo implements Serializable {
+
+    private static final long serialVersionUID = 1969725091044874463L;
```

Comment:
unused

Reply:
Java officials strongly recommend that all classes to be serialized explicitly declare the serialVersionUID division to avoid deserialization in JVM.